cmd/geth/config.go

```diff
@@ -203,6 +203,12 @@ func applyMetricConfig(ctx *cli.Context, cfg *gethConfig) {
 	if ctx.GlobalIsSet(utils.MetricsEnabledExpensiveFlag.Name) {
 		cfg.Metrics.EnabledExpensive = ctx.GlobalBool(utils.MetricsEnabledExpensiveFlag.Name)
 	}
+	if ctx.GlobalIsSet(utils.MetricsEnabledRecordIOFlag.Name) {
+		cfg.Metrics.EnabledExpensive = ctx.GlobalBool(utils.MetricsEnabledRecordIOFlag.Name)
```

Comment:
why reuse  `cfg.Metrics.EnabledExpensive`, since it will introduce other expensive metrics other than IO metrics?

Reply:
it should be EnableIORecord，but it is not useful now , I has tried to record I/O cost of one block using EnableIORecord, but  then I found out that this is not necessary ， I will delete the relevant code

Reply:
updated